mkdocs.yml

```diff
@@ -0,0 +1,115 @@
+site_name: DIDComm.org
+theme:
+  name: material
+  logo: didcomm-logo.png
+  favicon: didcomm-logo.png
+  features:
+    - navigation.tabs
+    - navigation.sections
+    - toc.integrate
+    - navigation.top
+    - search.suggest
+    - search.highlight
+    - content.tabs.link
+    - content.code.annotation
+    - content.code.copy
+  language: en
+  palette:
+    - scheme: default
+      toggle:
+        icon: material/toggle-switch-off-outline 
+        name: Switch to dark mode
+      primary: teal
+      accent: purple 
+    - scheme: slate 
+      toggle:
+        icon: material/toggle-switch
+        name: Switch to light mode    
+      primary: teal
+      accent: lime
+
+extra:
+  social:
+    - icon: fontawesome/brands/github-alt
+      link: https://github.com/TelegramSam/didcomm.org
+
+markdown_extensions:
+  - pymdownx.highlight:
+      anchor_linenums: true
+  - pymdownx.inlinehilite
+  - pymdownx.snippets
+  - admonition
+  - pymdownx.arithmatex:
+      generic: true
+  - footnotes
+  - pymdownx.details
+  - pymdownx.superfences
+  - pymdownx.mark
+  - attr_list
+  - pymdownx.emoji:
+      emoji_index: !!python/name:material.extensions.emoji.twemoji
+      emoji_generator: !!python/name:material.extensions.emoji.to_svg
+
+copyright: |
+  &copy; 2024 <a href="https://github.com/TelegramSam/didcomm.org"  target="_blank" rel="noopener">DIDComm.org</a>
+
+nav:
+  - Home: index.md
+  - Protocols:
+    - protocols/index.md
+    - Action Menu:
```

Comment:
The URI of the protocols should be at the base of the site. You can see how this is done on the published version of the original gatsby version: https://didcomm.org

For example, didcomm.org/protocols/action-menu/1.0/readme.md should be didcomm.org/action-menu/1.0/